src/drivers/angular2.js

```diff
@@ -0,0 +1,99 @@
+import { omit, replaceObject } from '../lib';
+/* eslint-disable new-cap, object-shorthand */
+
+
+export let angular2 = {
+
+    global() {
+        return (window.ng && window.ng.core && window.ng.platformBrowser) ? 
+        {
+            Component: window.ng.core.Component,
+            NgModule: window.ng.core.NgModule,
+            ElementRef: window.ng.core.ElementRef,
+            NgZone: window.ng.core.NgZone,
+            BrowserModule: window.ng.platformBrowser.BrowserModule
+        } 
+        : false;
+    },
+
+    register(xcomponent, { Component, NgModule, ElementRef, NgZone, BrowserModule }) {
+
+        xcomponent.log('initializing angular2 component');
+        
+        const getBindingMetadata = () => {
```

Comment:
The only reason I kept it like that is that I was thinking of declaring outputs (event emitters) in a later feature based on type of the prop is function. But maybe we can worry about that later.